.ado/templates/e2e-testing-android.yml

```diff
@@ -39,3 +45,9 @@ steps:
       platform: 'android'
       buildArtifacts: variables['task.Build.status']
       directory: $(Build.SourcesDirectory)/apps/E2E
+
+  - script: |
+      yarn appium driver uninstall uiautomator2
```

Comment:
Do we _need_ to uninstall the drivers? We could skip this right?
cc - @samuelfreiberg

Reply:
No, we don't necessarily need to, assuming the VM fully resets itself between tests. I figure it's a good practice to clean up state before exiting the script, though.

Reply:
Alright, asked because we have not followed this for other setup templates